CHANGELOG.md

```diff
@@ -11,12 +11,21 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [2.3.9]
+
+### Fixed
+
+- Make sure that the top N best params files are cached in a separate
```

Comment:
This description seems no longer accurate (no more separate directory)

Reply:
fixed